frontend/components/card/CardConnect.vue

```diff
@@ -9,21 +9,19 @@
       >
         <Icon
           v-if="userIsAdmin && !editModeEnabled"
-          @click="toggleEditMode"
           name="bi:pencil-square"
           size="1.2em"
+          @click="toggleEditMode"
```

Comment:
I was thinking that the `@` attributes could be below conditionals? Was something that came to mind when looking at things last night.

Reply:
I can go through and change these with #151?

Reply:
What do you think? I don't have very much experience with Vue projects so I am not sure. To my guts, I think putting `@` attributes at the very bottom seems fine.

Reply:
We talked it over in the dev sync and folks were generally on board with having `@` attributes towards the top so the component's usage is prioritized and so we have only one focus point of the `aria-label` at the end :)